c/feature-dictionary.asciidoc

```diff
@@ -2,6 +2,14 @@
 // Creative Commons Attribution 4.0 International License; see
 // http://creativecommons.org/licenses/by/4.0/
 
+// “generic” opencl_c_feature_name
+ifdef::backend-html5[]
+:opencl_c_feature_name: pass:q[`\__opencl_c_<wbr><em>feature_<wbr>name</em>`]
```

Comment:
I don't think we want to include the `<em>` and `</em>` tags here, since it causes the HTML output to be _italicized_.

Reply:
That was actually the objective, although I should have been more explicit about the it in the commit message 8-)

Since  `feature_name` is not actually the string that appears in the feature macro, but a placeholder, I believe it would be better to make it typographically distinct.

I'm actually only now noticing that further down `__opencl_c_<name>` is used instead, using `<>` to mark the replaceable part. I think we should uniformize these. Shall I go with the `<>` idea?

Reply:
Using `<>` is more distinctive to me than italics, so I'd unify on `__opencl_c_<name>`.